tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
+        "target": "es2015",
```

Comment:
nitpick 1: for node 10+, the target is [es2018](https://node.green)

Reply:
Done. This makes sense, the SDK minimum node version is 10.x